### ISSUE TITLE
Warning: Unknown props 

### ISSUE THREAD

**Issue by sonwh98** [STRML/react-grid-layout#290]
I get the following error when using data-grid attribute on version 0.12.7

```
react.inc.js:20209 Warning: Unknown props `minConstraints`, `maxConstraints`, `onResizeStop`, `onResizeStart`, `onResize`, `handleSize`, `lockAspectRatio` on <div> tag. Remove these props from the element. For details, see https://fb.me/react-unknown-prop
    in div (created by reagent8)
    in e (created by e)
    in DraggableCore (created by e)
    in e (created by ReactGridLayout)
    in div (created by ReactGridLayout)
    in ReactGridLayout (created by reagent8)
    in div (created by reagent8)
    in reagent8 (created by koala.eucalyptus.content)
```

However, when use _grid, it works and layouts properly but I get this warning:

```
react.inc.js:20209 Warning: Unknown props `_grid`, `minConstraints`, `maxConstraints`, `onResizeStop`, `onResizeStart`, `onResize`, `handleSize`, `lockAspectRatio` on <div> tag. Remove these props from the element. For details, see https://fb.me/react-unknown-prop
    in div (created by reagent8)
    in e (created by e)
    in DraggableCore (created by e)
    in e (created by ReactGridLayout)
    in div (created by ReactGridLayout)
    in ReactGridLayout (created by reagent8)
    in div (created by reagent8)
    in reagent8 (created by koala.eucalyptus.content)
```

I am using this via ClojureScript but that should not be a factor. Anyone know what the problem is?

**Comment by ajmajma** [comment]
Bump! Having same issue using react 0.14.8, react-grid-layout 0.12.7.

**Comment by mcottingham** [comment]
I am also seeing this issue with react 15.2.0 and RGL 0.13.0.

I only see this warning when I try to add the `measureBeforeMount={true}` prop to my `ResponsiveReactGridLayout` component.

Code snippet is as follows:
`const ResponsiveReactGridLayout = WidthProvider(Responsive);`
```
<ResponsiveReactGridLayout
    measureBeforeMount={true} 
    verticalCompact={false}
    breakpoints={{lg: 1200, sm:800, xs: 600}}
    cols={{lg: 5, sm: 4, xs: 2}}
    onLayoutChange={this.layoutChanged.bind(this)}
    onResizeStop={this.onResizeStop.bind(this)}>
    {dashboardItems}
</ResponsiveReactGridLayout>
```

Error message:

> Warning: Unknown props `verticalCompact`, `measureBeforeMount`, `breakpoints`, `onLayoutChange`, `onResizeStop`, `mounted` on \<div\> tag. Remove these props from the element. For details, see https://fb.me/react-unknown-prop

I thought that this might be something wrong with my initialization of the responsive react grid layout so I tried flipping the default flag in the RGL source, but the same error is thrown.

**Comment by aaronz** [comment]
@mcottingham I got the same issue, the source code shows if measureBeforeMount then return div with these props and finally gets this error. Have you figured out a way to resolve this?
`render() {
    if (this.props.measureBeforeMount && !this.state.mounted) return <div {...this.props} {...this.state} />;
    return <ComposedComponent {...this.props} {...this.state} />;
  }`

**Comment by cp** [comment]
@aaronz @mcottingham BTW, these issues should be fixed in RGL 0.13.3 by this PR here https://github.com/STRML/react-grid-layout/pull/335
